.github/workflows/projects.yml

```diff
@@ -0,0 +1,20 @@
+name: Auto Assign to Project(s)
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I'd rather only add this for the specific step that needs it, to avoid exposing it to any other steps added in the future.